libraries/c_sdk/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -251,12 +251,21 @@
 /**
  * @brief Default config for Maximum Number of MQTT CONNECTIONS.
  * This config can be specified by the application based on number of MQTT
- * connections needed.
+ * connections needed. Maximum number of subscriptions that shim can hold is 2^7.
```

Comment:
you can put the actual number here which is 128

Reply:
This is at a wrong place. The comment is for number of connections

Reply:
Addressed